lua/cyberdream/extensions/neogit.lua

```diff
@@ -5,28 +5,34 @@ local M = {}
 --- @param t CyberdreamPalette
 function M.get(opts, t)
     opts = opts or {}
+    local neogit = require("neogit")
```

Comment:
We need to check if `neogit` can be loaded before we try to require it. If you don't have the plugin installed, it breaks cyberdream:

```
Failed to run `config` for cyberdream.nvim

vim/_editor.lua:0: /home/scott/git/nix/nvim/init.lua..nvim_exec2() called at /home/scott/git/nix/nvim/init.lua:0../home/scott/git/cyberdream.nvim/colors/cyberdream.lua: Vim(colorscheme):E5113: Error while calling lua chunk: ...git/cyberdream.nvim/lua/cyberdream/extensions/neogit.lua:8: module 'neogit' not found:
	no field package.preload['neogit']
```

```suggestion
    if not package.loaded["neogit"] then
        return {}
    end
    local neogit = require("neogit")
```